src/index.js

```diff
@@ -0,0 +1,52 @@
+import './style.css';
+
+import { initializeTasks, createNewTask } from './modules/tasks.js';
+import {
+  markTaskComplete,
+  markTaskIncomplete,
+  clearCompletedTasks,
+} from './modules/status.js';
+
+const input = document.querySelector('#new-task-input');
+const submitButton = document.querySelector('.add');
+
+initializeTasks();
+
+submitButton.onclick = (e) => {
+  e.preventDefault();
+  if (input.value !== '') {
+    createNewTask(input.value);
+    input.value = '';
+  }
+};
```

Comment:
✔️ I really appreciate the efficiency of this submit event code. But,
> I would recommend to make your code even more generalized to use the submit event (onsubmit) on the form it self (as a general). Rather than a single button (idknow).